controllers/rentals.js

```diff
@@ -0,0 +1,97 @@
+var Rentals = require("../models/rentals");
+
+var RentalsController = {
+
+  getCurrentRentals: function (request, response, next) {
+    Rentals.find_current(request.params.customer_id, function(error, rentals) {
+      if(error) {
+        var err = new Error
+        err.status = 500;
+        err.error = "Error retrieving customer's current rentals."
+        response.json(err)
+      } else {
+        response.json(rentals)
+      }
+    });
+  },
+
+  getRentalHistory: function (request, response, next) {
+    Rentals.find_history(request.params.customer_id, function(error, rentals) {
+      if(error) {
+        var err = new Error
+        err.status = 500;
+        err.error = "Error retrieving customer's rental history."
+        response.json(err)
```

Comment:
A neat way to handle what @hougland is suggesting is creating an error `middleware`. See: https://derickbailey.com/2014/09/06/proper-error-handling-in-expressjs-route-handlers/ for an example